cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -306,6 +312,13 @@ instance FromJSON PartialNodeConfiguration where
                     "Invalid value for 'MempoolCapacityBytesOverride'.  \
                     \Expecting byte count or NoOverride.  Value was: " <> show invalid
               Nothing -> return Nothing
+
+      parseLedgerDBBackend v = do
+        maybeString :: Maybe String <- v .:? "LedgerDBBackend"
+        case maybeString of
+           Just "MEM" -> return $ Just InMemory
+           _          -> return $ Just LMDB
```

Comment:
Maybe an ill-formed flag should fail instead of default to `LMDB`

Reply:
This is now the behavior. I think now it is more ergonomic